community/kernel/src/main/java/org/neo4j/kernel/internal/Version.java

```diff
@@ -170,6 +149,20 @@ public static void main( String[] args )
     }
 
     static final String KERNEL_ARTIFACT_ID = "neo4j-kernel";
-    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID,
-            Version.class.getPackage().getImplementationVersion() );
+    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID, getMavenVersion() );
```

Comment:
Because the previous implementation was broken. It was always null. This actually works.